examples/demo/src/categories/CategoryList.tsx

```diff
@@ -5,7 +5,7 @@ import {
     RecordContextProvider,
     useListContext,
 } from 'react-admin';
-import inflection from 'inflection';
+import * as inflection from 'inflection';
```

Comment:
I think you should only import the utility you need for each script, to enable tree-shaking.

Reply:
I was thinking the same thing I'll work on it 🤝